addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -279,6 +281,31 @@ export class SelectionPlugin extends Plugin {
                     return this.setSelection({ anchorNode, anchorOffset, focusNode, focusOffset });
                 }
             }
+            if (this.correctDoubleClick) {
+                this.correctDoubleClick = false;
+                const { anchorNode, anchorOffset, focusNode } = this.activeSelection;
+                const anchorElement = closestElement(anchorNode);
+                // Allow editing the text of a link after "double click" on the last word of said link.
+                // This is done by corecting the selection focus inside of the link
+                if (
+                    anchorElement.tagName === "A" &&
```

Comment:
Maybe out of scope, but is it enough to do this for links only? Aren't there other elements for which we'd want this, like button, code, strong, ...?

Also, OCD detail but we usually use `nodeName`, especially when we know it's an element.